examples/waterwave.py

```diff
@@ -0,0 +1,108 @@
+# waterwave.py
+
+from random import randrange, random
+import taichi as ti
+import numpy as np
+
+ti.init(arch=ti.opengl)
+
+bg = 'misc/test_image.png'
+bg = 'misc/test_small.png'
+bg = ti.imread(bg, 3)
+bg = np.sum(bg, axis=2)
+bg = bg.astype(np.float32) / (256 * 3)
+bg = np.flip(bg, axis=0)
+bg = bg.transpose()
```

Comment:
How about using a simple checkerboard pattern here so that people can focus on the simulation & refraction? This way we also get rid of the image files.